workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -154,6 +170,158 @@ private void OnCommandResponse(CommandResponseOp op)
             specificDispatcher.OnCommandResponse(op);
         }
 
+        private void OnCreateEntityResponse(CreateEntityResponseOp op)
+        {
+            if (!createEntityStorage.CommandRequestsInFlight.TryGetValue(op.RequestId.Id, out var requestBundle))
```

Comment:
I think this is a good example of when `out var` is bad. I can't see what type this is and it's hard when reading code further down to see where this is created.